src/basis/tracker.js

```diff
@@ -444,65 +686,16 @@ function addDispatcher(dispatcher, events, transformer){
   return true;
 }
 
-function loadMap(map){
-  if (!map)
-  {
-    /** @cut */ basis.dev.warn(namespace + '.loadMap(): Wrong value for map');
-    return;
-  }
-
-  for (var key in map)
-  {
-    var eventsMap = map[key];
-
-    if (!eventsMap)
-    {
-      /** @cut */ basis.dev.warn(namespace + '.loadMap(): Value of map should be an object for path: ' + key);
-      continue;
-    }
-
-    var selector = parsePath(key);
-    for (var events in eventsMap)
-    {
-      var data = eventsMap[events];
-
-      events.trim().split(/\s+/).forEach(function(eventName){
-        registrateSelector(selector, eventName, data);
-      });
-    }
-  }
-};
-
-function getInfo(path){
-  var result = [];
-
-  if (typeof path == 'string')
-    path = parsePath(path);
-
-  for (var key in selectorMap)
-    if (isPathMatchSelector(path, selectorMap[key].selector))
-      result.push.apply(result, selectorMap[key].map(function(item){
-        return basis.object.extend({
-          selector: selectorMap[key].selector,
-          selectorStr: selectorMap[key].selectorStr
-        }, item);
-      }));
-
-  return result.length ? result : false;
-}
-
-
 module.exports = {
-  parsePath: parsePath,
-  parseRole: parseRole,
-  stringifyPath: stringifyPath,
-  stringifyRole: stringifyRole,
+  /** @cut */ parseRole: parseRole,
```

Comment:
Tests absence is not the reason to make API private. It should be provate when there are no useful use cases in prod. Also if it's doesn't use anywhere there is no reason to add something to export.